api/test_history.go

```diff
@@ -83,10 +83,10 @@ func testHistoryHandler(w http.ResponseWriter, r *http.Request) {
 
 	for _, run := range runs {
 
-		_, ok := testsByBrowser[run.Browser]
+		_, ok := testsByBrowser[run.BrowserName]
```

Comment:
nit: move the type declaration to the top of the file, `type Subtest map[string]string` and `type Browser map[string][]Subtest`

Reply:
done 👍